content/en/docs/configuration/acme/dns01/google.md

```diff
@@ -25,10 +25,12 @@ DNS01 challenge. To enable this, a GCP service account must be created with the
 > Console.
 
 ```bash
-$ export PROJECT_ID=myproject-id
+$ PROJECT_ID=myproject-id
```

Comment:
Why is export dropped here?

Reply:
Because it serves no purpose. Nowhere is `PROJECT_ID` used at an environment variable. It is just used as a shell variable.

Reply:
I just noticed later on, good point!